### DISCUSSION TITLE
RTC

### DISCUSSION THREAD

**Discussion by iceyman** [sanni/cartreader#776] [Hardware]
I see someone made a pcb to combine clock generator together but can’t find schematic, is there a way to manually add a RTC?

**Comment by Ancyker** [comment]
I am about to be ordering a board from JLCPCB to test adding RTC. Expecting some results towards the end of the month.

![image](https://user-images.githubusercontent.com/7105333/232629842-ef9c2f7f-6911-4d85-b651-d55ad9dc66e3.png)

**Reply by iceyman** [reply]
Cant wait to put an order in for these pcb
